smartcontract/storage/xsharddb.go

```diff
@@ -81,6 +81,14 @@ func (self *XShardDB) SetLockedAddress(addrs []comm.Address) {
 	self.cacheDB.put(common.XSHARD_KEY_LOCKED_ADDRESS, nil, sink.Bytes())
 }
 
+func (self *XShardDB) SetLockedKeys(keys [][]byte) {
+
```

Comment:
need sort those keys to avoid indeterminacy, because they are generated from map.